Resources/views/Collector/db.html.twig

```diff
@@ -426,7 +486,9 @@
             }
 
             header.setAttribute('data-sort-direction', (-1*direction).toString());
-            header.children[0].innerHTML = direction > 0 ? '<span class="text-muted">&#9650;</span>' : '<span class="text-muted">&#9660;</span>';
+            header.children[0].innerHTML = direction > 0
+                ? `<span class="sort-icon">{{ include('@Doctrine/Collector/arrow-narrow-up.svg') }}</span>`
```

Comment:
Would be nice to know why are we replacing UTF-8 character with SVG, which is larger and harder to change.

Reply:
I think because the UTF-8 characters don't look nice. But I also didn't really like the arrows, so I leave this to someone with actual UX/UI skills.